sh_legshoot.lua

```diff
@@ -0,0 +1,28 @@
+PLUGIN.name = "Legs wyjebnik"
+PLUGIN.author = "Lechu2375"
+PLUGIN.description = "Now hitting right in the legs causes injures."
+PLUGIN.license =  "MIT not for use on Kaktusownia"
+if SERVER then
+    local legs = {
+        [HITGROUP_LEFTLEG] = true,
+        [HITGROUP_RIGHTLEG] = true
+    }
+    local resFactions = {  --table with factions enums that should be resistant to these effects
+        [FACTION_OTA] = true
```

Comment:
This should probably be made a comment since you can't be sure that users will have `FACTION_OTA` in their schema